Makefile

```diff
@@ -1,15 +1,23 @@
 #
 # Makefile to run the business analytics software using metabase
 #
+#||: means or true, and prevents the target from crashing if there is an error
+#
 
 help :
-	@echo ""                                                                           ; \
-	echo "Useful targets in this business-analytics Makefile:"                         ; \
-	echo "- up           : run docker-compose up"  								                     ; \
-	echo "- up-detach    : run docker-compose up --detach"   				                   ; \
-	echo "- clean        : run rm ./database-backups/*.sql"   				                 ; \
-	echo "- down         : run docker-compose down"                                    ; \
-	echo ""
+	@echo ""                                                         ; \
+	echo "Useful targets in this business-analytics Makefile:"       ; \
+	echo "- up: run docker-compose up"  								       			 ; \
+	echo ""																													 ; \
+	echo "- up-detach: run docker-compose up --detach"   				     ; \
+	echo ""																								  			   ; \
+	echo "- remove-database-backups: run rm ./database-backups/*.sql"; \
+	echo ""																													 ; \
+	echo "- clean: removes metabase log file"  											 ; \
+	echo "         and removes the mysql volume"    								 ; \
+	echo ""																													 ; \
+	echo "- down: run docker-compose down"                           ; \
+	echo ""																													 ; \
```

Comment:
The command separator and line continuation that are at the end of the lines above are now all over the place.
I prefer them neatly lined up myself, but they should be either neatly lined up OR just follow the command on the line w/ a single space if you don't want to line them up.